package/Dockerfile.lighthouse-agent

```diff
@@ -1,20 +1,20 @@
 ARG BASE_BRANCH
 ARG SOURCE=/go/src/github.com/submariner-io/lighthouse
-ARG TARGETPLATFORM=linux/amd64
 
-FROM quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH} AS builder
+FROM --platform=${BUILDPLATFORM} quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH} AS builder
 ARG SOURCE
 ARG TARGETPLATFORM
 
 COPY . ${SOURCE}
 
-RUN make -C ${SOURCE} LOCAL_BUILD=1 bin/lighthouse-agent
+RUN make -C ${SOURCE} LOCAL_BUILD=1 bin/${TARGETPLATFORM}/lighthouse-agent
 
-FROM scratch
+FROM --platform=${TARGETPLATFORM} scratch
```

Comment:
This seems redundant per [this blog post]?

> For this stage, we will not use --platform in the FROM definition. We could write FROM --platform=$TARGETPLATFORM but that is the default value for all builds stages anyway so using a flag is redundant.

[this blog post]: https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

Reply:
It’s redundant per the official docs too, but I included it for balance. I thought it preferable to be explicit, so that readers wouldn’t have to wonder what the platform is when it’s not specified.